### PR TITLE
Add grype parser

### PR DESCRIPTION
**Pull request by dtbaum**
Adding support for Grype scanner reports, see https://plugins.jenkins.io/grypescanner/ and https://github.com/anchore/grype.

This is a continuation of [analysis-model/pull/935 PR](https://github.com/jenkinsci/analysis-model/pull/935).

### CODE REVIEW

File: plugin/src/main/java/io/jenkins/plugins/analysis/warnings/Grype.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import org.jenkinsci.Symbol;
+import hudson.Extension;
+import io.jenkins.plugins.analysis.core.model.AnalysisModelParser;
+
+/**
+ * Provides parser for grype reports.
+ */
+public class Grype extends AnalysisModelParser {
+    private static final long serialVersionUID = -7721519870683487886L;
+    private static final String ID = "grypescanner";
```
Comment (uhafner) [inline plugin/src/main/java/io/jenkins/plugins/analysis/warnings/Grype.java:13]: This ID is used also as URL. So if you want it to be shorter (e.g. "grype") then we need to change it in analysis-model.

**Review by uhafner — CHANGES_REQUESTED** [review]
Please add:

- merge with #1564  (when green)
- a new test case in `ParsersITest`
- optional: an SVG icon

File: plugin/src/main/java/io/jenkins/plugins/analysis/warnings/Grype.java (hunk 1 above)
Comment (dtbaum) [reply plugin/src/main/java/io/jenkins/plugins/analysis/warnings/Grype.java:13]: I have changed id to "grype" in [PR 936](https://github.com/jenkinsci/analysis-model/pull/936).  I will update the ID in the current PR once the change in in [PR 936](https://github.com/jenkinsci/analysis-model/pull/936) is released. After that, I will also update the new version of the model API, 11.5.1, here. Thanks!

**Comment by dtbaum** [comment]
> Please add:
> 
> * merge with [Bump io.jenkins.plugins:analysis-model-api from 11.3.0 to 11.5.0 #1564](https://github.com/jenkinsci/warnings-ng-plugin/pull/1564)  (when green)
> * a new test case in `ParsersITest`
> * optional: an SVG icon

Done!

File: plugin/src/main/java/io/jenkins/plugins/analysis/warnings/Grype.java (hunk 1 above)
Comment (dtbaum) [reply plugin/src/main/java/io/jenkins/plugins/analysis/warnings/Grype.java]: As discussed, id is changed to "grype", API dependency set to 11.5.1. Ready for new version of warnings-ng-plugin :-)

**Review by uhafner — APPROVED** [review]

**Pull request merged**